README.md

```diff
@@ -4,6 +4,13 @@ tax-year
 
 This is a UK tax year utility. Therefore, it uses the `Europe/London` timezone.
 
+Change History
```

Comment:
Please change to the ticket text, namely:

v4.0 - 05 Dec 23
- add support for Play 2.9 & Play 3.0, noting Play 2.8 is still supported
- update Scala 2.13 version
- update library dependencies
- add 'Change History' section to README

v3.3.0 - 01 Aug 23
- add README instructions

v3.2.0 - 06 May 22
- update Scala version

v2.0.0 - 07 Apr 22
- drop redundant scalatestplus-play dependency